test/acceptance/cli-test.js

```diff
@@ -195,13 +236,12 @@ describe('ember-template-lint executable', function() {
 
     describe('with --json param', function() {
       it('should print valid JSON string with errors', function() {
+        setProjectConfigForErrors();
         let result = run(['.', '--json'], {
-          cwd: './test/fixtures/with-errors',
+          cwd: project.path('.'),
```

Comment:
Seems like `cwd` is _always_ going to be `project.path()`, can we just make that the default (removing the repeated manual `cwd: project.path()` calls all over the place)

Reply:
💯